File: chap4/Horaire.c
// Ces programmes sont sous licence CeCILL-B V1.
#include <stdio.h>

static void TirerUnTrait ()
{
   printf("\n");
   printf("-----------------------------------------------\n");
   printf("\n");
}

int main() 
{
   printf("Le vol en direction de ");
   printf("Tokyo");
   printf(" décolera à ");
   printf("9h00");
   TirerUnTrait();

   printf("Le vol en direction de ");
   printf("Sydney");
   printf(" décolera à ");
   printf("9h30");
   TirerUnTrait();

   printf("Le vol en direction de ");
   printf("Toulouse");
   printf(" décolera à ");
   printf("9h45");
   TirerUnTrait();
}

File: chap4/Horaire.ml
(* Ces programmes sont sous licence CeCILL-B V1. *)

let tirerUnTrait () =
  print_newline ();
  print_string "-----------------------------------------------";
  print_newline ();
  print_newline ()

let _ =
  print_string "Le vol en direction de ";
  print_string "Tokyo";
  print_string " décolera à ";
  print_string "9h00";
  tirerUnTrait();

  print_string "Le vol en direction de ";
  print_string "Sydney";
  print_string " décolera à ";
  print_string "9h30";
  tirerUnTrait();

  print_string "Le vol en direction de ";
  print_string "Toulouse";
  print_string " décolera à ";
  print_string "9h45";
  tirerUnTrait()
